test/unit/busterbunny-raiseEvents-input-validation-tests.js

```diff
@@ -0,0 +1,92 @@
+describe('busterbunny.js raiseEvents', function() {
```

Comment:
Since only negative scenarios are tested in this suite, please relabel the describe to something more appropriate. Maybe `busterbunny.js raiseEvents validation tests`.